### ISSUE TITLE
[Bug]: [perf-nightly] Milvus collection flush raise an error"collection name should not be empty"

### ISSUE THREAD

**Issue by jingkl** [milvus-io/milvus#27090]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.3.0-20230914-c41be660 
- Deployment mode(standalone or cluster):cluster
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

server:
```

perf-query-169465800-1-42-9965-etcd-0                             1/1     Running     0                16m     10.104.12.67    4am-node17   <none>           <none>
perf-query-169465800-1-42-9965-etcd-1                             1/1     Running     0                16m     10.104.4.186    4am-node11   <none>           <none>
perf-query-169465800-1-42-9965-etcd-2                             1/1     Running     0                16m     10.104.18.153   4am-node25   <none>           <none>
perf-query-169465800-1-42-9965-milvus-datacoord-7c9c799bb6h4z8d   1/1     Running     0                16m     10.104.9.229    4am-node14   <none>           <none>
perf-query-169465800-1-42-9965-milvus-datanode-56886766b6-rzskn   1/1     Running     3 (3m ago)       16m     10.104.21.114   4am-node24   <none>           <none>
perf-query-169465800-1-42-9965-milvus-indexcoord-5964d7d5ffgdbs   1/1     Running     0                16m     10.104.21.113   4am-node24   <none>           <none>
perf-query-169465800-1-42-9965-milvus-indexnode-77788fc9d7s46mw   1/1     Running     0                16m     10.104.13.85    4am-node16   <none>           <none>
perf-query-169465800-1-42-9965-milvus-proxy-b9fd7dff4-qxsmr       1/1     Running     3 (3m40s ago)    16m     10.104.9.230    4am-node14   <none>           <none>
perf-query-169465800-1-42-9965-milvus-querycoord-78c67c49c5qmlg   1/1     Running     3 (2m51s ago)    16m     10.104.12.40    4am-node17   <none>           <none>
perf-query-169465800-1-42-9965-milvus-querynode-6dd6469f7chh7xw   1/1     Running     1 (11m ago)      16m     10.104.9.231    4am-node14   <none>           <none>
perf-query-169465800-1-42-9965-milvus-rootcoord-5b7c4b5bc9kqcs5   1/1     Running     2 (7m10s ago)    16m     10.104.18.122   4am-node25   <none>           <none>
perf-query-169465800-1-42-9965-minio-0                            1/1     Running     0                16m     10.104.12.74    4am-node17   <none>           <none>
perf-query-169465800-1-42-9965-minio-1                            1/1     Running     0                16m     10.104.4.188    4am-node11   <none>           <none>
perf-query-169465800-1-42-9965-minio-2                            1/1     Running     0                16m     10.104.18.154   4am-node25   <none>           <none>
perf-query-169465800-1-42-9965-minio-3                            1/1     Running     0                16m     10.104.19.90    4am-node28   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-bookie-0                    1/1     Running     0                16m     10.104.18.150   4am-node25   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-bookie-1                    1/1     Running     0                16m     10.104.12.65    4am-node17   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-bookie-2                    1/1     Running     0                16m     10.104.19.88    4am-node28   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-bookie-init-qnpg8           0/1     Completed   0                16m     10.104.13.79    4am-node16   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-broker-0                    1/1     Running     0                16m     10.104.12.42    4am-node17   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-proxy-0                     1/1     Running     0                16m     10.104.4.170    4am-node11   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-pulsar-init-6znc6           0/1     Completed   0                16m     10.104.18.121   4am-node25   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-recovery-0                  1/1     Running     0                16m     10.104.18.123   4am-node25   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-zookeeper-0                 1/1     Running     0                16m     10.104.13.97    4am-node16   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-zookeeper-1                 1/1     Running     0                13m     10.104.17.217   4am-node23   <none>           <none>
perf-query-169465800-1-42-9965-pulsar-zookeeper-2                 1/1     Running     0                6m28s   10.104.23.13    4am-node27   <none>           <none> (cli_client.py:132)
```

client log:

```
[2023-09-14 04:47:39,882 -  INFO - fouram]: [Base] Total time of insert: 35.6378s, average number of vector bars inserted per second: 28060.0935, average time to insert 50000 vectors per time: 1.7819s (base.py:396)
[2023-09-14 04:47:39,882 -  INFO - fouram]: [Base] Start flush collection fouram_q52X0CJt (base.py:292)
[2023-09-14 04:47:39,890 - ERROR - fouram]: RPC error: [get_flush_state], <MilvusException: (code=1, message=collection name should not be empty)>, <Time:{'RPC start': '2023-09-14 04:47:39.889138', 'RPC error': '2023-09-14 04:47:39.889957'}> (decorators.py:108)
[2023-09-14 04:47:39,890 - ERROR - fouram]: RPC error: [flush], <MilvusException: (code=1, message=collection name should not be empty)>, <Time:{'RPC start': '2023-09-14 04:47:39.882900', 'RPC error': '2023-09-14 04:47:39.890568'}> (decorators.py:108)
[2023-09-14 04:47:39,891 - ERROR - fouram]: (api_response) : <MilvusException: (code=1, message=collection name should not be empty)> (api_request.py:53)
[2023-09-14 04:47:39,891 - ERROR - fouram]: [CheckFunc] flush request check failed, response:<MilvusException: (code=1, message=collection name should not be empty)> (func_check.py:52)
```

### Steps To Reproduce

```markdown
1. create a collection 
        2. build hnsw index on vector column
        3. insert sift1m 128d of vectors
        4. flush collection -> raise error
```

### Anything else?

[2023-09-14 04:46:51,924 -  INFO - fouram]: [check_params] scene_concurrent_locust required params: {'dataset_params': {'metric_type': 'L2', 'dim': 128, 'dataset_name': 'sift', 'dataset_size': '1m', 'ni_per': 50000}, 'load_params': {}, 'search_params': {}, 'resource_groups_params': {'reset': False}, 'index_params': {'index_type': 'HNSW', 'index_param': {'M': 8, 'efConstruction': 200}}, 'concurrent_params': {'concurrent_number': [1, 20, 50, 100], 'during_time': 600, 'interval': 20}, 'concurrent_tasks': [{'type': 'query', 'weight': 1, 'params': {'expr': 'id in [1, 100, 1000]'}}]} (params_check.py:31)

**Comment by yanliang567** [comment]
confirmed with @czs007 , this was fixed by pr #27063, please help to verify 
/assign @jingkl 
/unassign @czs007

**Comment by jingkl** [comment]
This issue no longer arises
image: master-20231004-6d4492d2
